charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -22,7 +22,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 39.4.0
+version: 39.4.1
```

Comment:
New feature should raise minor version.

Please also sign your commits.

Reply:
I forgot to sign off the last commit. Fixed. 
What do you mean _raise the minor version_ - I bumped it. And it's not a new feature, it's a kind of bugfix.